.github/workflows/release-published.yml

```
@@ -442,7 +441,7 @@ jobs:
           remoteDir: "/xml/ns/dbchangelog/"
 
   release-docker:
-    if: always()
+    if: false  # TEMP: skip for v5.0.2 recovery - already completed
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)